docs/installation/linux.md

```diff
@@ -363,28 +395,26 @@ sudo rm -rf /usr/local/bin/powershell /usr/local/microsoft/powershell
 To uninstall the additional PowerShell paths (such as the user profile path)
 please see the [paths][paths] section below in this document
 and remove the desired the paths with `sudo rm`.
+(Note: this is not necessary if you installed with Homebrew.)
 
 [paths]:#paths
 
-## OpenSSL
+### OpenSSL on macOS
+
+On macOS, .NET Core requires Homebrew's OpenSSL
+because the "OpenSSL" system libraries on macOS are not OpenSSL,
+as Apple deprecated OpenSSL in favor of their own libraries.
+This requirement is not a hard requirement for all of PowerShell.
+However, most networking functions (such as `Invoke-WebRequest`)
+do require OpenSSL to work properly.
 
-Also install [Homebrew's OpenSSL][openssl]:
+The easiest fix is to install [Homebrew's OpenSSL][openssl]:
```

Comment:
This is auto-included in powershell brew formula (it's a dependency I think)
Worth pointing out here that it's not required as a separate step with brew.